### ISSUE TITLE
Parser crash when parsing byte string literals?

### ISSUE THREAD

**Issue by mattbasta** [prettier/plugin-python#76]
Stack trace:

```
[error] ../../tmp/pinecast/accounts/migrations/0001_initial.py: Error: Traceback (most recent call last):
[error]   File "/opt/prettier-python/vendor/python/astexport.py", line 123, in <module>
[error]     main()
[error]   File "/opt/prettier-python/vendor/python/astexport.py", line 118, in main
[error]     json = export_json(tree, True)
[error]   File "/opt/prettier-python/vendor/python/astexport.py", line 21, in export_json
[error]     separators=(",", ": ") if pretty_print else (",", ":")
[error]   File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/__init__.py", line 238, in dumps
[error]     **kw).encode(obj)
[error]   File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/encoder.py", line 201, in encode
[error]     chunks = list(chunks)
[error]   File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/encoder.py", line 430, in _iterencode
[error]     yield from _iterencode_dict(o, _current_indent_level)
[error]   File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/encoder.py", line 404, in _iterencode_dict
[error]     yield from chunks
[error]   File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/encoder.py", line 325, in _iterencode_list
[error]     yield from chunks
[error]   File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/encoder.py", line 404, in _iterencode_dict
[error]     yield from chunks
[error]   File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/encoder.py", line 325, in _iterencode_list
[error]     yield from chunks
[error]   File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/encoder.py", line 404, in _iterencode_dict
[error]     yield from chunks
[error]   File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/encoder.py", line 404, in _iterencode_dict
[error]     yield from chunks
[error]   File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/encoder.py", line 325, in _iterencode_list
[error]     yield from chunks
[error]   File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/encoder.py", line 404, in _iterencode_dict
[error]     yield from chunks
[error]   File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/encoder.py", line 325, in _iterencode_list
[error]     yield from chunks
[error]   File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/encoder.py", line 404, in _iterencode_dict
[error]     yield from chunks
[error]   File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/encoder.py", line 404, in _iterencode_dict
[error]     yield from chunks
[error]   File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/encoder.py", line 325, in _iterencode_list
[error]     yield from chunks
[error]   File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/encoder.py", line 404, in _iterencode_dict
[error]     yield from chunks
[error]   File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/encoder.py", line 325, in _iterencode_list
[error]     yield from chunks
[error]   File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/encoder.py", line 404, in _iterencode_dict
[error]     yield from chunks
[error]   File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/encoder.py", line 325, in _iterencode_list
[error]     yield from chunks
[error]   File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/encoder.py", line 404, in _iterencode_dict
[error]     yield from chunks
[error]   File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/encoder.py", line 404, in _iterencode_dict
[error]     yield from chunks
[error]   File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/encoder.py", line 325, in _iterencode_list
[error]     yield from chunks
[error]   File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/encoder.py", line 404, in _iterencode_dict
[error]     yield from chunks
[error]   File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/encoder.py", line 325, in _iterencode_list
[error]     yield from chunks
[error]   File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/encoder.py", line 404, in _iterencode_dict
[error]     yield from chunks
[error]   File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/encoder.py", line 437, in _iterencode
[error]     o = _default(o)
[error]   File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/encoder.py", line 180, in default
[error]     o.__class__.__name__)
[error] TypeError: Object of type 'bytes' is not JSON serializable
[error]
[error]     at parseText (/opt/prettier-python/src/parser.js:18:11)
[error]     at Object.parse (/opt/prettier-python/src/parser.js:26:27)
[error]     at Object.parse (/opt/prettier-python/node_modules/prettier/src/main/parser.js:75:19)
[error]     at formatWithCursor (/opt/prettier-python/node_modules/prettier/index.js:101:25)
[error]     at Object.formatWithCursor (/opt/prettier-python/node_modules/prettier/index.js:402:12)
[error]     at format (/opt/prettier-python/node_modules/prettier/src/cli/util.js:157:19)
[error]     at eachFilename (/opt/prettier-python/node_modules/prettier/src/cli/util.js:380:16)
[error]     at filePaths.forEach.filePath (/opt/prettier-python/node_modules/prettier/src/cli/util.js:322:7)
[error]     at Array.forEach (<anonymous>)
[error]     at eachFilename (/opt/prettier-python/node_modules/prettier/src/cli/util.js:321:15)
[error]     at Object.formatFiles (/opt/prettier-python/node_modules/prettier/src/cli/util.js:338:3)
[error]     at Object.run (/opt/prettier-python/node_modules/prettier/src/cli/index.js:60:12)
[error]     at Object.<anonymous> (/opt/prettier-python/node_modules/prettier/bin/prettier.js:5:23)
[error]     at Module._compile (module.js:660:30)
[error]     at Object.Module._extensions..js (module.js:671:10)
[error]     at Module.load (module.js:573:32)
```

While parsing this file:

https://github.com/Pinecast/pinecast/blob/master/accounts/migrations/0001_initial.py

Running Python 3.6.4

**Comment by FuegoFro** [comment]
Yeah I've run into this issue as well. It looks like a simple fix to make it not crash would be to add some method like this to `DictExportVisitor` in `astexport.py`

```python3
    def visit_field_Bytes_s(self, val):
        return str(val)
```

However, the actual autoformatter currently doesn't do anything to try to handle `Bytes` AST nodes properly, so it would just print out a warning about them and leave those tokens unchanged. Which, granted, is better than crashing ;)

**Comment by patrick91** [comment]
Fixed in #91
